assets/MicrosoftUpdate/RegisterMicrosoftUpdate.ps1

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
```

Comment:
Since we are shipping this file and using it for installation, should we sign it as well?

Reply:
PS1 files are all signed:  https://github.com/PowerShell/PowerShell/blob/f38e181fa901430ee93933cfad4916bc8942734b/tools/releaseBuild/azureDevOps/templates/windows-packaging.yml#L91